lang/Lib/gflanguages/data/languages/arb_Arab.textproto

```diff
@@ -8,13 +8,13 @@ sample_text {
   masthead_full: "يولد"
   masthead_partial: "جم"
   styles: "يولد جميع الناس أحرارًا متساوين في الكرامة"
-  tester: "يولد جميع الناس أحرارًا متساوين في الكرامة والحقوق. وقد وهبوا عقلاً"
+  tester: "يولد جميع الناس أحرارًا متساوين في الكرامة والحقوق  وقد وهبوا عقلاً"
```

Comment:
This replaces space period with space space, but HTML will turn the double into a single, so I think its better to replace period with nothing (removing it)

Reply:
Agree, U+06D4 is the right fix. All Noto Arabics support that.

Reply:
Urgh, hang on. There is one font in the library (Noto Kufi Arabic) which has the Arabic full stop but not the Latin full stop. There are 21 Arabic fonts in the library which have the Latin full stop but not the Arabic full stop. So using U+06D4 is good news for Noto but bad news for the rest of GF.

Reply:
replaced with "۔" (Arabic full stop). PTAL

Reply:
> using U+06D4 is good news for Noto but bad news for the rest of GF

Currently these files are only used for the Noto specimen pages, so that's fine for now. 

The longer term fix is to add Latin full stop to the Noto web fonts and then change this back to Latin.